lib/puppet/util/filetype.rb

```diff
@@ -191,13 +191,21 @@ def path=(user)
 
     # Read a specific @path's cron tab.
     def read
+      unless Puppet::Util.uid(@path)
+        Puppet.debug _("The %{path} user does not exist. Treating their crontab file as empty in case Puppet creates them in the middle of the run.") % { path: @path }
```

Comment:
How about the debug message here? Originally an error message was printed, but this isn't really an error case as the workflow of "Create new user => create their crontab" is an acceptable scenario.

Reply:
Talked to @branan -- debug is fine here.